index.html

```diff
@@ -293,62 +293,70 @@
 
-Data is <dfn data-lt="pseudonymous data|pseudonymity">pseudonymous</dfn> when:
+Data is <dfn data-lt="pseudonymous data|pseudonymity">pseudonymous</dfn> when there are
+strict controls that prevent the re-identification of [=people=] described by the data
+except for a well-defined set of [=purposes=].
+
+Different situations involving [=pseudonymous data=] will require different controls. For instance,
+if the [=pseudonymous data=] is <em>only</em> being processed by one [=party=], typical controls
+include making sure that the [=identifiers=] used in the data are unique to that dataset, that
+any person (e.g. an employee of the [=party=]) with access to the [=data=] is barred (e.g. based on
+legal terms) from sharing the [=data=] further, and that technical measures exist to prevent
+re-identification or the joining of different data sets involving this [=data=], notably against
+timing or k-anonymity attacks.
+
+In general, the goal is to ensure that [=pseudonymous data=] is used in a manner that provides a
+viable degree of oversight and accountability such that technical and procedural means to guarantee
+the maintenance of pseudonymity are preserved.
+
+This is more difficult when the [=pseudonymous data=] is shared between several [=parties=]. In such
+cases, typical controls would include making sure that
 
-* the identifiers used in the data are under the direct and exclusive control of
-  the [=first party=]; and
+* the [=identifiers=] used in the [=data=] are under the direct and exclusive control of
+  the [=first party=];
 
-* when these identifiers are shared with a [=third party=], they are made unique
+* when these [=identifiers=] are shared with a [=third party=], they are made unique
   to that [=third party=] such that if they are shared with more than one
-  [=third party=] these cannot then match them up with one another; and
+  [=third party=] these cannot then match them up with one another;
 
-* there is a strong level of confidence that no [=third party=] can match them
-  to any data other than that obtained through interactions with the [=first
-  party=]; and
+* there is a strong level of confidence that no [=third party=] can match the [=data=]
+  with any data other than that obtained through interactions with the [=first
+  party=];
 
-* any [=third party=] receiving such identifiers is barred (eg. based on legal
-  terms) from sharing them or the related data further; and
+* any [=third party=] receiving such [=data=] is barred (eg. based on legal terms)
+  from sharing it further;
 
 * technical measures exist to prevent re-identification or the joining of
-  different data sets involving these identifiers, notably against timing or
+  different data sets involving this [=data=], notably against timing or
   k-anonymity attacks; and
 
 * there exist contractual terms between the [=first party=] and [=third party=]
   describing the limited [=purpose=] for which the data is being shared.
```

Comment:
I'm not sure it makes sense to go into this much detail about the design of controls around the "additional information" that can re-identify the data. What are the principles that drive the design of this list?